kuma/users/jinja2/users/email/payment_received/plain.ltxt

```diff
@@ -1,10 +1,20 @@
 {% autoescape false -%}
-{{ _('Hello from MDN!') }}
+Thank you.
 
-{{ invoice_pdf }}
+You supported MDN with a ${{ dollar_amount }} monthly subscription.
+
+Amount paid: ${{ dollar_amount }}.00
+Payment method: {{ credit_card_brand }}
+
+Invoice number: {{ invoice_number }}
+Date of issue: {{ payment_date }}
```

Comment:
I went with that initially but kept getting a syntax error and abandoned it. Just looked at it again and realized that I tried django templating filter syntax, instead of jinja's 🙈